src/components/Sum.jsx

```diff
@@ -1 +1,21 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
Wrap if you have more than 2 params or line more that 80
```suggestion
export const Sum = ({ a, b }) => (
```